src/Base/RestAPI/Controllers/SdgController.php

```diff
@@ -0,0 +1,197 @@
+	 *
+	 * @return array
+	 */
+	public function pagination( $query, $results = [] ) : array {
+
+		$page = $query->get('paged');
+        $page = 0 == $page ? 1 : $page;
+
+		return [
+			'count'    => (int) $query->found_posts,
+			'next'     => ( $query->max_num_pages > $page ? add_query_arg( 'page', ( $page + 1 ), get_rest_url( null, '/owc/pdc/v1/sdg/' ) ) : null ), 
+			'previous' => ( $page > 1 ? add_query_arg( 'page', ( $page - 1 ), get_rest_url( null, '/owc/pdc/v1/sdg/' ) ) : null ),
+			'results' => $results,
+		];
+	}
+
+	public static function metaQuery(array $parametersFromRequest = []): array
+	{
+		$query = [
+			'meta_query' => [
+				[
+					'key'     => '_owc_pdc_active',
+					'value'   => '1',
+					'compare' => '=',
+				],
+				[
+					'key'     => '_owc_enrichment_send_data_to_sdg',
+					'value'   => '1',
+					'compare' => '=',
+				],
+				[
+					'key'     => '_owc_enrichment_version',
+					'compare' => 'EXISTS',
+				]
+			]
+		];
+
+		if(! empty($parametersFromRequest)){
+			if ( isset( $parametersFromRequest['lang'] ) ) {
+				$lang  = ! empty( $parametersFromRequest['lang'] ) ? esc_attr( $parametersFromRequest['lang'] ) : 'nl';
+				$query = self::metaLanguageQuery( $lang, $query );
+			}
+
+			if ( isset( $parametersFromRequest['modified_before'] ) ) {
+				$query['date_query'][] = array(
+					'before'  => $parametersFromRequest['modified_before'],
+					'column' => 'post_modified',
+				);
+			}
+
+			if ( isset( $parametersFromRequest['modified_after'] ) ) {
+				$query['date_query'][] = array(
+					'after'  => $parametersFromRequest['modified_after'],
+					'column' => 'post_modified',
+				);
+			}
+
+			if ( isset( $parametersFromRequest['upnUri'] ) ) {
+				$upl = esc_attr( $parametersFromRequest['upnUri'] );
+				$query['meta_query'][] = [
+					'key' => '_owc_pdc_upl_resource',
+					'value' => $upl,
+					'compare' => '=',
+				];
+			}
+
+			if ( isset( $parametersFromRequest['doelgroep'] ) ) {
+				$audience = esc_attr( $parametersFromRequest['doelgroep'] );
+				$query['meta_query'][] = [
+					'key' => '_owc_enrichment_audience',
+					'value' => $audience,
+					'compare' => '=',
+				];
+			}
+		}
+
+		return $query;
+	}
+
+	protected static function metaLanguageQuery(string $lang, array $query): array
+	{
+		if($lang === 'nl') {
+			$query['meta_query'][] = [
+				'relation' => 'OR',
+				[
+					'key'     => '_owc_pdc-item-language',
+					'value'   => 'nl',
+					'compare' => '=',
+				],
+				[
+					'key'     => '_owc_pdc-item-language',
+					'value'   => 'nl',
+					'compare' => 'NOT EXISTS',
+				]
+			];
+
+			return $query;
+		}
+
+		$query['meta_query'][] = [
+			[
+				'key'     => '_owc_pdc-item-language',
+				'value'   => $lang,
+				'compare' => '=',
+			]
+		];
+
+		return $query;
+	}
+
+	public function arguments()
+	{
+		$args      = [];
+		$args['modified_before'] = [
+			'description' => esc_html__('Get all items modified before a specific date.', 'pdc-base'),

```

Comment:
We moeten niet vergeten om de vertalingen nog toe te voegen maar dat kan ik wel doen als mijn teksten duidelijk zijn en vertaal kunnen worden. Ik zet deze taak in de TODO.md.